### PR TITLE
Fix decompile errors

### PR DESCRIPTION
**Pull request by hYdos**
E

### CODE REVIEW

**Comment by hYdos** [comment]
Our Work

File: patches/net/minecraft/entity/passive/BeeEntity.java.patch [hunk 1]
```diff
@@ -1,5 +1,59 @@
 --- a/net/minecraft/entity/passive/BeeEntity.java
 +++ b/net/minecraft/entity/passive/BeeEntity.java
+@@ -587,7 +587,7 @@
+ 
+    class EnterHiveGoal extends BeeEntity.NotAngryGoal {
+       private EnterHiveGoal() {
+-         super(null);
```
Comment (ramidzkh) [inline patches/net/minecraft/entity/passive/BeeEntity.java.patch:7]: Please verify with the bytecode

File: patches/net/minecraft/entity/passive/FoxEntity.java.patch [hunk 2]
```diff
@@ -25,6 +34,24 @@
  
        for(int j = 0; j < 6; ++j) {
           double g = f == 0.0D ? 0.0D : d * (double)((float)j / 6.0F);
+@@ -973,7 +972,7 @@
+       private int counter;
+ 
+       public SitDownAndLookAroundGoal() {
+-         super(null);
```
Comment (ramidzkh) [inline patches/net/minecraft/entity/passive/FoxEntity.java.patch:41]: Please verify with the bytecode

File: patches/net/minecraft/item/DyeItem.java.patch [hunk 3]
```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/item/DyeItem.java
 +++ b/net/minecraft/item/DyeItem.java
+@@ -35,7 +35,7 @@
+       return ActionResult.PASS;
+    }
+ 
+-   public DyeColor getColor() {
++   public static DyeColor getColor() {
```
Comment (ramidzkh) [inline patches/net/minecraft/item/DyeItem.java.patch]: What?

**Review by BoogieMonster1O1 — APPROVED** [review]
concern

File: patches/net/minecraft/entity/mob/SpiderEntity.java.patch [hunk 4]
```diff
@@ -1,5 +1,16 @@
 --- a/net/minecraft/entity/mob/SpiderEntity.java
 +++ b/net/minecraft/entity/mob/SpiderEntity.java
+@@ -136,8 +136,8 @@
+    }
+ 
+    @Nullable
+-   public EntityData initialize(ServerWorldAccess world, LocalDifficulty difficulty, SpawnReason spawnReason, @Nullable EntityData entityData, @Nullable CompoundTag entityTag) {
+-      EntityData entityData = super.initialize(world, difficulty, spawnReason, entityData, entityTag);
++   public EntityData initialize(ServerWorldAccess world, LocalDifficulty difficulty, SpawnReason spawnReason, @Nullable EntityData edata, @Nullable CompoundTag entityTag) {
```
Comment (ramidzkh) [inline patches/net/minecraft/entity/mob/SpiderEntity.java.patch:9]: Please verify with the bytecode

File: patches/net/minecraft/entity/mob/ZombieEntity.java.patch [hunk 5]
```diff
@@ -1,5 +1,16 @@
 --- a/net/minecraft/entity/mob/ZombieEntity.java
 +++ b/net/minecraft/entity/mob/ZombieEntity.java
+@@ -399,8 +399,8 @@
+    }
+ 
+    @Nullable
+-   public EntityData initialize(ServerWorldAccess world, LocalDifficulty difficulty, SpawnReason spawnReason, @Nullable EntityData entityData, @Nullable CompoundTag entityTag) {
+-      EntityData entityData = super.initialize(world, difficulty, spawnReason, entityData, entityTag);
++   public EntityData initialize(ServerWorldAccess world, LocalDifficulty difficulty, SpawnReason spawnReason, @Nullable EntityData edata, @Nullable CompoundTag entityTag) {
++      EntityData entityData = super.initialize(world, difficulty, spawnReason, edata, entityTag);
```
Comment (ramidzkh) [inline patches/net/minecraft/entity/mob/ZombieEntity.java.patch:10]: Please verify with the bytecode

File: patches/net/minecraft/world/PortalForcer.java.patch [hunk 6]
```diff
@@ -1,15 +1,33 @@
 --- a/net/minecraft/world/PortalForcer.java
 +++ b/net/minecraft/world/PortalForcer.java
-@@ -33,7 +33,7 @@
-       pointOfInterestStorage.preloadChunks(this.world, blockPos, i);
+@@ -27,14 +27,14 @@
+       this.world = world;
+    }
+ 
+-   public Optional<class_5459.class_5460> method_30483(BlockPos blockPos, boolean bl) {
++   public Optional<class_5459.class_5460> method_30483(BlockPos blockPos1, boolean bl) {
```
Comment (ramidzkh) [inline patches/net/minecraft/world/PortalForcer.java.patch:8]: Please verify with the bytecode

File: patches/net/minecraft/world/level/storage/LevelStorage.java.patch [hunk 7]
```diff
@@ -1,5 +1,32 @@
 --- a/net/minecraft/world/level/storage/LevelStorage.java
 +++ b/net/minecraft/world/level/storage/LevelStorage.java
+@@ -107,20 +107,20 @@
+       var10002.getClass();
+       return Pair.of(dataResult.resultOrPartial(Util.method_29188("WorldGenSettings: ", var10002::error)).orElseGet(() -> {
+          DataResult var10000 = RegistryLookupCodec.of(Registry.DIMENSION_TYPE_KEY).codec().parse(dynamic3);
+-         Logger var10002 = LOGGER;
```
Comment (ramidzkh) [inline patches/net/minecraft/world/level/storage/LevelStorage.java.patch:11]: Undo these changes, or fix them

**Review by ramidzkh — CHANGES_REQUESTED** [review]

File: patches/net/minecraft/entity/passive/SheepEntity.java.patch [hunk 8]
```diff
@@ -1,6 +1,18 @@
...
-@@ -202,7 +202,7 @@
+@@ -37,10 +37,7 @@
+ import net.minecraft.entity.mob.MobEntity;
+ import net.minecraft.entity.player.PlayerEntity;
+ import net.minecraft.inventory.CraftingInventory;
+-import net.minecraft.item.DyeItem;
+-import net.minecraft.item.ItemConvertible;
+-import net.minecraft.item.ItemStack;
+-import net.minecraft.item.Items;
++import net.minecraft.item.*;
```
Comment (ramidzkh) [inline patches/net/minecraft/entity/passive/SheepEntity.java.patch]: Unneeded change

**Review by ramidzkh — APPROVED** [review]
